ubiquity-os-marketplace/ubiquity-os-kernel-telegram
# Create on assignment

Resolves #6

- `createchat` runs on `issues.assigned`
- we look for an existing chat using the `node_id` and skip if it exists

QA:

view the payload event for `issues.assigned` and confirm chat via logs or [join here](https://t.me/+orF-n39-xo42MWZk)

Chat creation:
https://github.com/ubq-testing/telegram--bot/actions/runs/11447419429/job/31848699310

Skipping creation if exists:
https://github.com/ubq-testing/telegram--bot/actions/runs/11447435712/job/31848754535

Cloudflare deployed and operational:
![image](https://github.com/user-attachments/assets/3ddb8e6c-21f2-4b2d-a4e2-8bac36fd2aba)

![image](https://github.com/user-attachments/assets/de78c513-5b82-450a-9385-41b70988066d)
